modules/admin_manual/pages/release_notes.adoc

```diff
@@ -18,7 +19,15 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
-== Changes in 10.1
+== Changes in 10.1.1
+
+ownCloud Server 10.1.1 is a hotfix follow-up release that takes care of an issue with loading updated apps. Instead of updating the app versions to their new values in the database, the old version value is written causing the process to repeat with every request. (https://github.com/owncloud/core/issues/34851)
```

Comment:
How about linking "an issue with loading updated apps" to https://github.com/owncloud/core/issues/34851, instead of writing the link separately?

Reply:
Good point, changed.